app/enterprise/1.3-x/db-encryption.md

```diff
@@ -49,6 +49,13 @@ export KONG_KEYRING_PRIVATE_KEY=/path/to/generated/key.pem
 
 All nodes in the Kong cluster should share the same `keyring_enabled` and `keyring_strategy` configuration values. Not every node needs to be provided the management RSA key pair, as that key pair is only used for backup and recovery processes. It does not need to be present for regular database read/write operations.
 
+Note that the user under which Kong worker processes run (as defined by the `nginx_user` Kong configuration option) must have read access to the public and private keys in order to be able to perform keyring export and import operations. We recommend restricting access to these files as tightly as possible. For example:
```

Comment:
```suggestion
The Kong worker processes run under the user you define in Kong's configuration property, `nginx_user`. Note that this user must have read access to the public and private keys in order to be able to perform keyring export and import operations. We recommend restricting access to these files as tightly as possible. For example:
```

Reply:
to me this reads oddly that we start the sentence discussing the kong user, with no context, followed by a hard stop. this is one thought- "the user must have read access..." vs "there is a user**.** it must have read access..."

will defer to you here

Reply:
I agree that it should have some context. I wanted to break apart the sentence since I found it hard to follow. Do you think this conveys enough detail?

```suggestion
Note that the user (defined by the `nginx_user` property) must have read access to the public and private keys in order to be able to perform keyring export and import operations. We recommend restricting access to these files as tightly as possible. For example:
```